data/autoyast_qam/15-common_base_installation.xml.ep

```diff
@@ -280,12 +280,17 @@
     % if ($get_var->('SCC_ADDONS') =~ m/\brt\b/) {
     <kernel>kernel-rt</kernel>
     % }
-    % if ($check_var->('VERSION', '15-SP3') or $check_var->('VERSION', '15-SP4')) {
     <packages config:type="list">
+      <!-- Workaround for bsc#1202234: [addon]-release packages are missing after autoyast installation. -->
```

Comment:
Isn't this workaround for 15-SP3 or 15-Sp3+ only ?

Reply:
No, seems sle12 can hit the same issue, please see the job:

https://openqa.suse.de/tests/overview?arch=&flavor=&machine=&test=mru-install-desktop-with-addons&modules=&module_re=&distri=sle&build=20230508-1&groupid=414#

The case is marked as softfail with my previous workaround at https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/17004